runWPT.sh

```diff
@@ -8,40 +8,51 @@ log() {
 }
 
 usage() {
-  log "Usage: [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] $0 [webdriver/tests/bidi/[...]]"
+  log "Usage: [CHROMEDRIVER=<true | false>] [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] $0 [webdriver/tests/bidi/[...]]"
 }
 
 if [[ $# -gt 0 && ("$1" == "-h" || "$1" == "--help") ]]; then
   usage
   exit 0
 fi
 
+# The path to the browser binary.
+readonly CHROME_BIN="${CHROME_BIN:-/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev}"
+
+# Whether to use Chromedriver with mapper.
+readonly CHROMEDRIVER="${CHROMEDRIVER:-false}"
+
 # Whether to start the server in headless or headful mode.
 readonly HEADLESS="${HEADLESS:-true}"
 
 # The path to the WPT manifest file.
 readonly MANIFEST="${MANIFEST:-MANIFEST.json}"
 
 # The browser product to test.
-readonly PRODUCT="${PRODUCT:-chromium}"
+readonly PRODUCT="${PRODUCT:-chrome}"
 
-# Whether to update the WPT expectations after running the tests.
-readonly UPDATE_EXPECTATIONS="${UPDATE_EXPECTATIONS:-true}"
+# Multiplier relative to standard test timeout to use.
+readonly TIMEOUT_MULTIPLIER="${TIMEOUT_MULTIPLIER:-8}"
 
-# The path to the browser binary.
-readonly WPT_BROWSER_PATH="${WPT_BROWSER_PATH:-/Applications/Google Chrome Dev.app/Contents/MacOS/Google Chrome Dev}"
+# Whether to update the WPT expectations after running the tests.
+readonly UPDATE_EXPECTATIONS="${UPDATE_EXPECTATIONS:-false}"
```

Comment:
False by default to play better with CI. Alternatively it could be true, but then we'd have to set it to false in CI. Either is fine.